src/Entity/Message.php

```diff
@@ -409,4 +410,16 @@ public function setLanguage($language) {
     $this->language = $language;
   }
 
+  /**
+   * Default value callback for 'uid' base field definition.
+   *
+   * @see ::baseFieldDefinitions()
```

Comment:
I haven't seen this partial `@see` in core - are you sure it's a valid format? Maybe better add the full path.